core/templates/dev/head/editor/EditorServices.js

```diff
@@ -59,7 +59,17 @@ oppia.factory('explorationData', [
             $log.info('Retrieved exploration data.');
             $log.info(response.data);
 
-            explorationData.data = response.data;
+            // FIXME: Update this value based on what exactly is being sent from
```

Comment:
Hi @jacobdavis , the backend implementation for this is not done yet. I'll update it as soon as that's done.